packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason.jsx

```diff
@@ -19,28 +19,28 @@ const initial_form = {
     'The platforms lack key features or functionality': false,
     'Customer service was unsatisfactory': false,
     "I'm deactivating my account for other reasons": false,
-    otherTradingPlatforms: '',
-    doToImprove: '',
+    other_trading_platforms: '',
+    do_to_improve: '',
 };
 
 const preparingReason = values => {
     let selected_reasons = selectedReasons(values)
         .map(val => val[0])
         .toString();
-    const is_other_trading_platform__has_value = !!values.otherTradingPlatforms.length;
-    const is_to_do_improve_has_value = !!values.doToImprove.length;
+    const is_other_trading_platform__has_value = !!values.other_trading_platforms.length;
+    const is_to_do_improve_has_value = !!values.do_to_improve.length;
```

Comment:
The issue is still there as you can copy and paste extra characters more that the character limit... please help to revise 🙏🏼